test/e2e/operator.go

```
@@ -329,6 +212,150 @@ func TestKCMRecovery(t *testing.T) {
+	name := names.SimpleNameGenerator.GenerateName("pdbtest-")
+	_, err = kubeClient.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+	},
+		metav1.CreateOptions{},
+	)
+	if err != nil {
+		t.Fatalf("could not create test namespace: %v", err)
+	}
+	defer kubeClient.CoreV1().Namespaces().Delete(ctx, name, metav1.DeleteOptions{})
```

golangci-lint (2.11.4)

[error] Error return value of `(k8s.io/client-go/kubernetes/typed/core/v1.NamespaceInterface).Delete` is not checked

(errcheck)